calculatorUICOOP/ViewModels/MainPageViewModel.cs

```diff
@@ -150,7 +150,7 @@ public void Equals()
                     DisplayContent = MathLogic.Multiply(_number1, _number2).ToString();
                     break;
                 case "/":
-                    if (DisplayContent != "0")
+                    if (decimal.Parse(DisplayContent) != 0.0m)
```

Comment:
Unsafe parse of  `DisplayContent`. This will throw an unhandled exception in the event any characters exist in the string.

Reply:
I have an idea.